screens/onboarding/res/String.js

```
@@ -0,0 +1,9 @@
+const String = {
```

Biome (1.9.4)

[error] Do not shadow the global "String" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)